component/testrunner/testrunner.json

```diff
@@ -161,7 +161,7 @@
         },
         "uris" :
         {
-          "add-nocache-param" : true
+          "add-nocache-param" : false
```

Comment:
You're right this should probably be a separate pull request and discussion; it snuck in here because I constantly find myself disabling it if I want to debug code in the unit or performance tests because (in Chrome at least) it means that the breakpoints are not saved from one run to another and it's difficult to find the code to set the breakpoint without running.

Personally I can't think of a time when the nocache param has ever been useful - AFAICR it was introduced because of Chrome's aggressive caching, and maybe Chrome's behaviour has changed since then?

I'm happy to back out this change and make a separate PR, although if no-one objects I'd just as happily leave it in :)